docs/appendices/release-notes/5.7.0.rst

```diff
@@ -94,6 +95,10 @@ Scalar and Aggregation Functions
 
 - Added the :ref:`scalar-reverse` scalar function.
 
+- `rebdiaz <https://github.com/rebdiaz>`_ added support for lower case ``yyyy``
+  format string to :ref:`to_char <scalar-to_char>` scalar function, which
+  behaves exactly the same as uppercase ``YYYY`` to match PostgreSQL behaviour.
+
```

Comment:
Hi @rebdiaz, I haven't found full name in your account, so using GH handle. 
Feel free to suggest a version with a full name if you want.